lib/importer.js

```diff
@@ -12,29 +12,39 @@ export function openImportDialog() {
   return dialog.showOpenDialog({
     title: "Open Quiver Notebook",
     properties: ["openFile"],
-    filters: [{ name: "Quiver Library", extensions: ["qvlibrary"] }],
+    filters: [
+      { name: "Quiver Library", extensions: ["qvlibrary", "qvnotebook"] }
+    ],
     defaultPath: app.getPath("home")
   });
 }
 
 export async function importNotebooksFromQuiverLibrary(files) {
   if (files.length !== 1) {
     inkdrop.notifications.addError("invalid file is selected.", {
-      detail: e.stack,
```

Comment:
`e` is `undefined`. It cause error when the user cancel the file dialog. (`files.length === 0` pattern)